docs/manual/manual_ifci_control.rst

```diff
@@ -0,0 +1,61 @@
+
+.. figure:: ../_static/manual_images/pulsing_propeller/Long_Packed_Control_Message.png
+    :align: center
+    :alt: Packed Control Message Description
+
+    Packed Control Message Description
+
+A general description of IQUART packets can be found in :ref:`manual_iquart`.
+
+Each control value is two bytes and can encode either a throttle value, an X value, or a Y value. These values are considered 'Raw' and represent a range of 0.0 to 1.0. The modules can read any individual control value in the Packed Control Message through a corresponding Control Value Index (CVI) value :ref:`IQUART Flight Controller Interface Client<ifci_message_table>`. For example if throttle_cvi is set to 2 and a packed control message has a control value in index 2 then the module will set its throttle based on the value at that control index. The same is true for x_cvi and y_cvi. The module's responses to control values are defined by settings in the :ref:`ESC Propeller Input Parser<epip_message_table>` and :ref:`Pulsing Rectangular Input Parser<prip_message_table>`.
+
+More information on setting up throttle response can be found in :ref:`manual_throttle`.
+
+The X and Y CVIs convert the raw 0.0 to 1.0 values to -1.0 to 1.0. The setting 'pulsing_voltage_mode' controls whether this value is considered a percentage of supply voltage or a percentage of the value defined by 'pulsing_voltage_limit'. When set to supply voltage mode -1.0 maps to a pulsing voltage on that axis of negative battery voltage and 1.0 maps to positive battery voltage. When set to voltage limit mode you must also set the 'pulsing_voltage_limit' parameter. This will set the maximum and minimum pulsing voltage. In this mode -1.0 will map to -'pulsing_voltage_limit' and 1.0 will map to 'pulsing_voltage_limit'. Between these values, the raw value is linearly mapped to the voltages in between.
+
+While this can work to control a single module connected to a single UART port on your flight controller, you can also connect multiple modules to a single UART port for control as a bus. An example of the connection is shown below.
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_serial_connection.png
+    :align: center
+    :alt: Bussed Serial Connection
+
+    Bussed Serial Connection
+
+This will work without changing the module IDs, but if telemetry is required you will need to set each module's module ID to a different value. This is done in the :ref:`system control client<sc_message_table>` with the 'module_id' parameter. Each module on the bus should have a unique module ID.
```

Comment:
I'd suggest clarifying what "work" means here. 

"This will allow you to control the module's using control values, but...". Or something in that vein